src/components/Common/SocialSidebar/SidebarElements.jsx

```diff
@@ -1,18 +1,30 @@
 import styled from "styled-components";
 
 export const SidebarContainer = styled.div`
-    position: sticky;
+    position: ${(props) => (props.$sidebarType === "feeds" ? "unset" : "sticky")};
+    top: ${(props) => (props.$sidebarType === "feeds" ? "unset" : "100px")};
     display: flex;
     flex-direction: column;
     justify-content: flex-start;
     gap: 10px;
-    top: 100px;
     width: 100%;
-    max-width: 200px;
+    max-width: ${(props) => (props.$sidebarType === "feeds" ? "unset" : "200px")};
     min-width: ${(props) => (props.$sidebarType === "explore" ? "310px" : "400px")};
     color: #fff;
     box-shadow: 0 4px 8px rgb(0 0 0 / 10%);
     border-radius: 10px;
+    margin: ${(props) => (props.$sidebarType === "feeds" ? "1rem 0" : "unset")};
+    padding: ${(props) => (props.$sidebarType === "feeds" ? "0 3rem" : "unset")};
+
+    @media screen and (width >= 950px) {
+        position: ${(props) => (props.$sidebarType === "feeds" ? "sticky" : "")};
+        top: ${(props) => (props.$sidebarType === "feeds" ? "100px" : "")};
+        max-width: ${(props) => (props.$sidebarType === "feeds" ? "500px" : "200px")};
+    }
+
+    @media screen and (width >= 400px) {
+        padding: ${(props) => (props.$sidebarType === "feeds" ? "0 1rem" : "")};
```

Comment:
review of @Tamir198 

I think that you can avoid writing the same code by writing a function for the styles (after all styled components are just functions).

Do you think something like this will work/ is needed :


```js

const commonStyles = css`
    display: flex;
    flex-direction: column;
    justify-content: flex-start;
    gap: 10px;
    width: 100%;
    color: #fff;
    box-shadow: 0 4px 8px rgb(0 0 0 / 10%);
    border-radius: 10px;
`;

export const SidebarContainer = styled.div`
    ${(props) => props.$sidebarType === "feeds" && css`
        ${commonStyles}
        position: unset;
        max-width: 200px;
        margin: 1rem 0;
        padding: 0 3rem;

        @media screen and (min-width: 950px) {
            position: sticky;
            top: 100px;
            max-width: 500px;
        }

        @media screen and (min-width: 400px) {
            padding: 0 1rem;
        }
    `}

    ${(props) => props.$sidebarType === "explore" && css`
        ${commonStyles}
        position: sticky;
        top: 100px;
        min-width: 400px;
    `}
`;

```